src/core/index.ts

```diff
@@ -58,28 +50,40 @@ const approveChange = async (
   }
 };
 
+/**
+ * Create a check in the pull request to indicate that the app
+ * has received and processed the pull request event.
+ *
+ * Note:
+ *
+ * - It can only be passing or not exist at all.
+ * - It is for indicating that the event is handled.
+ * - It indicates nothing about whether the PR is approved.
+ *
+ * @param context The request context created by Probot core.
+ * @param startTime The timestamp that this run started.
+ */
 const createPassingStatus = async (
   context: Context,
   startTime: string,
-  endTime: string,
 ): Promise<void> => {
   const statusOptions: Octokit.RequestOptions &
     Octokit.ChecksCreateParams = context.repo({
-      "completed_at": endTime,
-      "conclusion": "success",
-      "head_sha": context.payload.pull_request.head.sha,
-      "name": APP_CHECK_NAME,
-      "output": {
-        "summary": "test",
-        "text": "test",
-        "title": "test",
+      completed_at: new Date().toISOString(),
+      conclusion: "success",
+      head_sha: context.payload.pull_request.head.sha,
+      name: APP_CHECK_NAME,
+      output: {
+        summary: "test",
+        text: "test",
+        title: "test",
       },
-      "request": {
-        "retries": 3,
-        "retryAfter": 3,
+      request: {
+        retries: 3,
+        retryAfter: 3,
       },
-      "started_at": startTime,
-      "status": "completed",
+      started_at: startTime,
```

Comment:
Codacy found an issue: [Identifier 'started_at' is not in camel case.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6092258)